src/detection/scripts/batch_submit.py

```diff
@@ -1,14 +1,23 @@
 #!/usr/bin/env python
 
-import argparse
 from os import environ
 
+import click
 import boto3
 
 s3_bucket = environ.get('S3_BUCKET')
 
 
-def submit(branch_name, command_args, attempts=3, cpu=False):
+@click.command()
+@click.argument('branch_name', help='Branch with code to run on Batch')
+@click.argument('command', help='Command in quotes to run on Batch')
+@click.option('--attempts', default=3, help='Number of times to retry job')
+@click.option('--cpu', is_flag=True, help='Use CPU EC2 instances')
+def batch_submit(branch_name, command_args, attempts=3, cpu=False):
```

Comment:
Is this still needed? Is this what gets run on the batch instance then?

Reply:
batch_submit is just the script that creates the Batch job to run the scripts remotely. I think it's still needed.

Reply:
Got it - it wasn't clear to me at first glance if this is what kicks off the batch job or if this is supposed to be the command that is running on batch.